docs/data-integrations/duckdb.mdx

```diff
@@ -28,7 +28,7 @@ The optional arguments are as follows:
 In order to make use of this handler and connect to the DuckDB database in MindsDB, the following syntax can be used:
 
 ```sql
-CREATE DATABASE duckdb_datasource
+CREATE DATABASE duckdb_datasource;
```

Comment:
The semicolon should be at the end of the statement and not in its first line. Please correct it.